shared/settings/notifications/index.desktop.js

```diff
@@ -7,10 +7,10 @@ import type {Props} from './index'
 
 function Notifications (props: Props) {
   return (
-    <Box style={{...globalStyles.flexBoxColumn}}>
+    <Box style={{...globalStyles.flexBoxColumn, padding: globalMargins.medium}}>
       <Text type='Header' style={{marginTop: globalMargins.xlarge}}>Email me:</Text>
       <Box style={globalStyles.flexBoxColumn}>
-        {props.settings.map(s => (
+        {!!props.settings && props.settings.map(s => (
```

Comment:
Good idea. I'll add this into the branch off this one (https://github.com/keybase/client/pull/4431) as this has some of that kind of logic already